css/index.css

```diff
@@ -99,18 +285,134 @@ p {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-evenly;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
 }
 
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
+    background-color: black;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
+   
 }
```

Comment:
the parent class .box can take care of all of the styling except for what you want to change in each individual box, in this case you only need to change the color of the boxes.